### PR TITLE
Improves Issue Report Template

### PR DESCRIPTION
**Pull request by ExcessiveUseOfCobblestone**
Purpose:

Fixes issues [like these](https://github.com/tgstation/tgstation/issues/40912#issuecomment-429643560).

Makes it clear WHERE TO WRITE for those not familiar with github markup so it doesn't get lost.

Headers allow you to link to a specific aspect of your issue report for slightly easier viewing and now don't go away in the issue report due to current markup.

### CODE REVIEW

**Comment by fludd12** [comment]
That explains *a lot*. Sorry for the weirdness.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
+<!--- **INCLUDE THE ROUND ID**
+If you discovered this issue from playing tgstation hosted servers:
+[Round ID]: # (It can be found in the Status panel or retrieved from https://atlantaned.space/statbus/round.php ! The round id let's us look up valuable information and logs for the round the bug happened.)-->
+
+## Testmerges:
 
-[Round ID]: # (If you discovered this issue from playing tgstation hosted servers:)
-[Round ID]: # (**INCLUDE THE ROUND ID**)
-[Round ID]: # (It can be found in the Status panel or retrieved from https://atlantaned.space/statbus/round.php ! The round id let's us look up valuable information and logs for the round the bug happened.)
+<!-- If you believe the issue to be caused by a test merge [OOC tab -> Show Server Revision], report it in the pull request's comment section instead. If None, feel free to remove this section. -->
```
Comment (MrDoomBringer) [inline .github/ISSUE_TEMPLATE/bug_report.md]: I would change this to remove the "if you believe part", as some testmerges might mess up seemingly unrelated functionality, and as such should always be listed

**Comment by Cyberboss** [comment]
nononono, if you're sure an issue is from an ongoing test merge it goes in said PR until it's merged

**Comment by ExcessiveUseOfCobblestone** [comment]
above commit addresses Cyber's comment.

**Comment by MrDoomBringer** [comment]
@Cyberboss o u right

**Pull request merged**